android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -411,11 +412,21 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
             Log.d(TAG, "create request to " + captive_portal_url + " for url: " + envoyUrl)
         }
 
+        // set up http cache dir
+        val cacheDir = File(applicationContext.cacheDir, envoyService + "-cache")
```

Comment:
I don't like using the service name... we could easily have e.g. more than one hysteria server in the future